### PR TITLE
survey_form_skeleton_and_style

### PR DESCRIPTION
**Pull request by MILINDI7**
## What?
a survey form

## Why?
As required by the project, the task is to build a simple survey form.

## How?
After creating a well organised repo, we have to build the form skeleton in Html and after style it in CSS, which is done in this featuring branch.

### CODE REVIEW

**Review by MILINDI7 — COMMENTED** [review]
Hi,
Great work, well done the project is fine working but there are some issues that are not addressed yet.
**Highlights**
- Linters run well
- perfect README

**Check comments for required changes**

File: index.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  
+<div class="choice">
+  <label id="checkbox">What would you like to see improved? (Check all that apply) </label><br>
+  <input type="checkbox" id="online" name="checkbox" value="Front-end Projects">
+ </div>
+
+<textarea id="comments" name="textarea" rows="4" cols="50" placeholder="enter your comments here..."></textarea><br><br>
+
+ <input id="submit" type="submit" value="Submit">
```
Comment (Abenezer-Tilahun) [inline index.html:59]: - [ ]     Good job so far 👏. Please provide an apparent reason WHY you are suggesting this change. You may consider providing links to some articles that explain that point very well 😉. 💡 Please edit the above comment which you have left before.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @MILINDI7 ,

Good job so far! 👏 👏
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️

## Highlights 💪

- [ ] Great use of GitHub review -- Inline comments feature
- [ ] Use polite phrases while giving reviews
- [ ] Provided a clear explanation of what changes to be made

Required Changes ♻️

- [ ] Your Pull Request should have a descriptive title telling you what changes are inside and a short summary describing introduced changes in more detail. Please try to add description in your Pull Request.

Check the comments under the review.
Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (MILINDI7) [reply index.html:59]: There is a little but significant difference between them. you might please check this link :https://www.tutorialspoint.com/What-is-difference-between-button-vs-input-type-button

**Review by Bondok6 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MILINDI7 🙋‍♂️,  

Your project is complete!  Congratulations!  🎉💯🌟 

- [OPTIONAL] You can include emojis to make your code review more human-friendly 👽



Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**